draft-illyes-cbcp.md

```diff
@@ -89,7 +101,24 @@ standard signals defined by
 [RFC9110](https://www.rfc-editor.org/rfc/rfc9110#name-server-error-5xx), preferably also
 additional heuristics such as relative response time of the server.
 
+As such, crawlers should log already visited URL and how many requests they sent to a resource 
+as well as the respective HTTP status codes in the responses,
+especially if error occur, to avoid repeatedly crawling the same source.
+
+Generally, crawlers should avoid multiple concurrent requests to the same ressources
+and use a crawling rate that is no higher than 1 request per second or as specified in
```

Comment:
This is not going to work and I would be very surprised if *any* crawler operator would agree to this. 1 QPS to most websites, including news like WSJ and CNN, but also ecomm like Amazon, would mean spending about 95 years to crawl them. Sites like Facebook add more than 86400 unique urls a day.

Reply:
Okay, you are right this doesn't work for large pages. It's just a recommendation I read somewhere but didn't think too much about. Maybe we can add some text that discusses a bit more that rate should be adopted based on provider knowledge and for some infrastructures a lower rate should be implemented. Is that something that would be realistic?

Reply:
Maybe we add something about this in [this issue](https://github.com/garyillyes/cbcp/issues/3)?

Reply:
Is this okay?
```suggestion
and should limit the crawling rate to avoid overload, if possible considering limits specified in
```